api/candlepin-api-spec.yaml

```diff
@@ -30,6 +30,196 @@ paths:
         default:
           $ref: '#/components/responses/default'
 
+  /consumers/{consumer_uuid}/guestids:
+    get:
+      description: Retrieves the list of a Consumer's Guests
+      tags:
+        - GuestIds
+      operationId: getGuestIds
+      x-java-response-container-type: Iterable
+      security: []
+      parameters:
+        - name: consumer_uuid
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: A successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/GuestIdDTOArrayElement'
+        default:
+          $ref: '#/components/responses/default'
+
+    put:
+      description: Updates the list of guests on a consumer. This method should work just like updating the consumer, except that it only updates guestIds. Eventually we should move all the logic here, and depricate updating guests through the consumer update.
```

Comment:
We should leave out the implementation details about moving and deprecating logic.

Reply:
Updated.